wiki/Medals/Unlock_requirements/Hush-hush/en.md

```diff
@@ -1,8 +1,8 @@
-# Unlock requirements for Hush-Hush medals
-
-*Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*
-
-- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
-- Medals can only be obtained on beatmaps that are either [Ranked](/wiki/Beatmap/Category#ranked), [Loved](/wiki/Beatmap/Category#loved), [Approved](/wiki/Beatmap/Category#approved) or [Qualified](/wiki/Beatmap/Category#qualified).
-- Map-specific medals, which can only be obtained on certain beatmaps, are always related to songs from [Featured Artists](/wiki/People/Featured_Artists). This is a convention which has been emphasized more recently, so some older medals are an exception to this rule.
-- Medals in the subcategory called "Hush-Hush (Expert)" do not allow usage of any [difficulty reduction](/wiki/Gameplay/Game_modifier#difficulty-reduction) or [special](/wiki/Gameplay/Game_modifier#special) mods.
\ No newline at end of file
+# Unlock requirements for Hush-Hush medals
+
+*Note: While some conditions for Hush-Hush medals are known, their solutions are intentionally omitted from this wiki to help preserve their secrecy.*
+
+- A medal only appears in the profile of [gamemodes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

Comment:
```suggestion
- On the osu! website, a medal appears in the user profile only for [game modes](/wiki/Game_mode) it can be obtained in. However, there are exceptions due to various reasons, such as star rating changes.
```

`game mode` are two words as far as I know.

Reply:
afaik there is no other gamemode profile apart from osu-web one, so what's even the point to state it

Reply:
this would also apply to the in-game profile (lazer) so yes it's a bit pointless to state